### PR TITLE
 Capture color exposure and white balance properties

### PR DESCRIPTION
**Pull request by shagren**
New properties added

### CODE REVIEW

File: pyk4a/capture.py [hunk 1]
```diff
@@ -50,6 +53,24 @@ def color_timestamp_usec(self) -> int:
...
+    def color_exposure_usec(self) -> int:
+        if self._color_exposure_usec is None:
+            value = k4a_module.color_image_get_exposure_usec(self._capture_handle)
+            if value == 0:
+                raise K4AException("Cannot read exposure from color image")
+            self._color_exposure_usec = value
+        return self._color_exposure_usec
+
+    @property
+    def color_white_balance(self) -> int:
```
Comment (shagren) [inline pyk4a/capture.py:66]: maybe `color_whitebalance` instead `color_white_balance` will be better?

Comment (lpasselin) [reply pyk4a/capture.py:66]: I prefer white_balance since it is what the SDK uses

File: pyk4a/capture.py [hunk 2]
```diff
@@ -50,6 +53,24 @@ def color_timestamp_usec(self) -> int:
             self.color
         return self._color_timestamp_usec
 
+    @property
+    def color_exposure_usec(self) -> int:
```
Comment (lpasselin) [inline pyk4a/capture.py:57]: Before we make one function for each, should we fetch all of the properties/metadata at the same time, in a single function?

Comment (lpasselin) [reply pyk4a/capture.py:57]: If those two are the only available properties, let's keep it that way.

File: pyk4a/capture.py (hunk 1 above)
Comment (shagren) [reply pyk4a/capture.py:66]: ok, lets's leave as is

**Comment by lpasselin** [comment]
@shagren thanks a lot for these PR once again!!

**Pull request merged**